indico/htdocs/sass/modules/abstracts/_abstract_page.scss

```diff
@@ -218,9 +231,24 @@
 }
 
 .abstract-comment-badges {
-    align-self: flex-end;
+    margin-right: auto;
```

Comment:
It's clearer here. 

---

Using `margin-*` to distribute elements means we lose the ability to use margin for actually adding margins. Example with long content.

![screenshot from 2016-11-22 16 10 12](https://cloud.githubusercontent.com/assets/716307/20529059/16880008-b0cf-11e6-8139-b0ba4e9b998f.png)

---

`flex-grow: 1` distributes the elements and allows us to keep using `margin-*` to define a margin.

![screenshot from 2016-11-22 16 10 36](https://cloud.githubusercontent.com/assets/716307/20529066/1dbceeb0-b0cf-11e6-92ff-c19f2fa52de3.png)